### ISSUE TITLE
Compile args "--fast-math=none" not working for U2Net

### ISSUE THREAD

**Issue by Udit1999** [aws/aws-neuron-sdk#394]
Getting error while compiling this particular model with "--fast-math=none". Here is the code snippet:
```
import torch
import torch_neuron
from u2net import U2NET

model = U2NET()
model.eval()
dummy_input = torch.randn(1,3,320,320)
model_neuron = torch_neuron.trace(model, 
                                  dummy_input,
                                 compiler_args = [ "--fast-math", "none"])

```
Kindly, find u2net.py file in [u2net.zip](https://github.com/aws/aws-neuron-sdk/files/8033719/u2net.zip).

Am I doing anything wrong?
Is there any other workaround to optimize this compilation for more accuracy?

```

Here are the error logs:02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:  An Internal Compiler Error has occurred
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]: ***************************************************************
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]: 
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]: Error message:  list index out of range
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]: 
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]: Error class:    IndexError
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]: Error location: Unknown
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]: Command line:   /newvol/home/ubuntu/aws_inferentia/venv/bin/neuron-cc compile /tmp/tmpr_l530mr/graph_def.pb --framework TENSORFLOW --pipeline compile SaveTemps --output /tmp/tmpr_l530mr/graph_def.neff --io-config '{"inputs": {"0:0": [[1, 3, 320, 320], "float32"]}, "outputs": ["Sigmoid:0", "Sigmoid_1:0", "Sigmoid_2:0", "Sigmoid_3:0", "Sigmoid_4:0", "Sigmoid_5:0", "Sigmoid_6:0"]}' --fast-math none --verbose 35
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]: 
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]: Internal details:
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/driver/CommandDriver.py", line 223, in neuroncc.driver.CommandDriver.CommandDriver.run
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/driver/commands/CompileCommand.py", line 525, in neuroncc.driver.commands.CompileCommand.CompileCommand.run
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/driver/Job.py", line 289, in neuroncc.driver.Job.SingleInputJob.run
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/driver/Pipeline.py", line 30, in neuroncc.driver.Pipeline.Pipeline.runSingleInput
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/driver/Job.py", line 289, in neuroncc.driver.Job.SingleInputJob.run
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/driver/Pipeline.py", line 30, in neuroncc.driver.Pipeline.Pipeline.runSingleInput
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/driver/Job.py", line 289, in neuroncc.driver.Job.SingleInputJob.run
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/driver/jobs/Frontend.py", line 427, in neuroncc.driver.jobs.Frontend.Frontend.runSingleInput
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/driver/jobs/Frontend.py", line 377, in neuroncc.driver.jobs.Frontend.Frontend.runTVMFrontend
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/driver/jobs/Frontend.py", line 378, in neuroncc.driver.jobs.Frontend.Frontend.runTVMFrontend
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/driver/jobs/Frontend.py", line 388, in neuroncc.driver.jobs.Frontend.Frontend.runTVMFrontend
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/driver/jobs/Frontend.py", line 358, in neuroncc.driver.jobs.Frontend.Frontend.runTVMFrontend.tensorize
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/starfish/penguin/Frontend.py", line 47, in neuroncc.starfish.penguin.Frontend.tensorizeRelay
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/starfish/penguin/Frontend.py", line 48, in neuroncc.starfish.penguin.Frontend.tensorizeRelay
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/starfish/penguin/Frontend.py", line 69, in neuroncc.starfish.penguin.Frontend.tensorizeRelay
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/starfish/penguin/Compile.py", line 240, in neuroncc.starfish.penguin.Compile.compile_cu
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/starfish/penguin/DotTransform.py", line 378, in neuroncc.starfish.penguin.DotTransform.PassManager.transformFunction
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/starfish/penguin/DotTransform.py", line 124, in neuroncc.starfish.penguin.DotTransform.DotTransform.runOnFunction
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/starfish/penguin/DotTransform.py", line 178, in neuroncc.starfish.penguin.DotTransform.DotTransform.run_with_exception_handling
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/starfish/penguin/DotTransform.py", line 160, in neuroncc.starfish.penguin.DotTransform.DotTransform.run_with_exception_handling
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/starfish/penguin/DotTransform.py", line 186, in neuroncc.starfish.penguin.DotTransform.DotTransform.timed_run_
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/starfish/penguin/DotTransform.py", line 188, in neuroncc.starfish.penguin.DotTransform.DotTransform.timed_run_
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/starfish/penguin/DotTransform.py", line 189, in neuroncc.starfish.penguin.DotTransform.DotTransform.timed_run_
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/starfish/penguin/DotTransform.py", line 206, in neuroncc.starfish.penguin.DotTransform.DotTransform.run_
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/starfish/penguin/DotTransform.py", line 207, in neuroncc.starfish.penguin.DotTransform.DotTransform.run_
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/starfish/penguin/DotTransform.py", line 287, in neuroncc.starfish.penguin.DotTransform.DotTransform.transformFunction
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/starfish/penguin/DotTransform.py", line 281, in neuroncc.starfish.penguin.DotTransform.DotTransform.runTransforms
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/starfish/penguin/targets/tonga/passes/TongaSizeTiling.py", line 1454, in neuroncc.starfish.penguin.targets.tonga.passes.TongaSizeTiling.TongaSizeTiling.transformStmts
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/starfish/penguin/targets/tonga/passes/TongaSizeTiling.py", line 1256, in neuroncc.starfish.penguin.targets.tonga.passes.TongaSizeTiling.TongaSizeTiling.transformDAG
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   File "neuroncc/starfish/penguin/targets/tonga/passes/TongaSizeTiling.py", line 644, in neuroncc.starfish.penguin.targets.tonga.passes.TongaSizeTiling.TongaSizeTiling.packInstsToMatMulMacro
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]: 
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]: Version information:
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   Neuron Compiler version 1.9.1.0+5f09e5c8e
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   HWM version 1.9.0.0-0
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   NEFF version Dynamic
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   TVM version 1.9.1.0+0
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   NumPy version 1.19.5
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   MXNet not available
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]:   TF not available
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]: 
02/11/2022 05:28:16 AM ERROR 7395 [neuron-cc]: Artifacts stored in: /tmp/tmpr_l530mr
```

**Comment by aws-zejdaj** [comment]
Hi Udit, thank you for sharing your model. I was able to reproduce the internal error in compiler. We are investigating the cause.

**Comment by aws-owinop** [comment]
Hi Udit1999, we are working on a fix for this issue and will share an update soon.

**Comment by Udit1999** [comment]
Hi @aws-owinop, thanks for the update.

**Comment by mrnikwaws** [comment]
Hi Udit1999,

we have identified the issue, and the fix has been made.  It will appear in a future release.  We'll update here once the fix is publicly available

**Comment by Udit1999** [comment]
Hi @mrnikwaws, Thanks for the update 👍

**Comment by Udit1999** [comment]
Hey aws team,

I tried with the latest SDK release ([1.18](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/release-notes/index.html#latest-neuron-release)), the same error still persists. Were you talking about the same release or planning to have another one soon with the current bug fixed?

**Comment by mrnikwaws** [comment]
Hi Udit1999,

When a fix is made it goes into a release pipeline to be tested along with other fixes and improvements.  Unfortunately that can't always be the release immediately following us identifying an issue.  In this case the fix has not yet made it out through the pipeline to the public neuron repositories.  We'll update this ticket when the fix is publicly released.

**Comment by aws-taylor** [comment]
Hello @Udit1999,

 We're making progress on this issue, but it has not made it into the release yet. We'll continue to update this issue as the fix is available.

**Comment by awsrjh** [comment]
Closing  and will be tracked in the roadmap item: 

https://github.com/aws/aws-neuron-sdk/issues/477
